extensions/2.0/Vendor/EXT_feature_metadata/examples/microcosm/microcosm.gltf

```diff
@@ -0,0 +1,286 @@
+{
+    "asset" : {
+        "generator" : "Khronos glTF Blender I/O v0.9.36",
+        "version" : "2.0"
+    },
+    "extensionsUsed": [
+        "EXT_feature_metadata"
+    ],
+    "extensions": {
+        "EXT_feature_metadata": {
+            "extras": {
+                "draftVersion": "0.0.0"
+            },
+            "featureClasses": {
+                "NDVI": {
+                    "name": "Normalized difference vegetation index",
+                    "properties": {
+                        "NDVI": {
+                            "valueType": "UINT8",
+                            "normalized": true
+                        }
+                    }
+                },
+                "LULC": {
+                    "name": "Landcover/Landuse",
+                    "properties": {
+                        "Name": {
+                            "name": "Name",
+                            "valueType": "STRING"
+                        },
+                        "Color": {
+                            "name": "Color",
+                            "elementType": "ARRAY",
+                            "valueType": "UINT8",
+                            "valueCount": 3
+                        }
+                    }
+                }
+            },
+            "featureCollections": {
+                "LandCover": {
+                    "featureClass": "LULC",
+                    "featureCount": 6,
+                    "properties": {
+                        "Name": {
+                            "elementCount": 256,
+                            "bufferView": 5,
+                            "offsetsBufferView": 6
+                        },
+                        "Color": {
+                            "elementCount": 256,
+                            "bufferView": 7
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "scene" : 0,
+    "scenes" : [
+        {
+            "name" : "Scene",
+            "nodes" : [
+                0,
+                1
+            ]
+        }
+    ],
+    "nodes" : [
+        {
+            "name" : "Camera",
+            "rotation" : [
+                0.483536034822464,
+                0.33687159419059753,
+                -0.20870360732078552,
+                0.7804827094078064
+            ],
+            "translation" : [
+                7.358891487121582,
+                4.958309173583984,
+                6.925790786743164
+            ]
+        },
+        {
+            "mesh" : 0,
+            "name" : "Terrain"
+        }
+    ],
+    "materials" : [
+        {
+            "doubleSided" : true,
```

Comment:
The material needs to include `extensions` and `EXT_feature_metadata` as well like in `texture-metadata.gltf`